modules/oe_newsroom_newsletter/tests/src/Traits/OeNewsroomNewsletterTrait.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Drupal\Tests\oe_newsroom_newsletter\Traits;
+
+use Drupal\oe_newsroom\OeNewsroom;
+use Drupal\oe_newsroom_newsletter\OeNewsroomNewsletter;
+
+/**
+ * Shared methods prepared to use in any test, if needed.
+ */
+trait OeNewsroomNewsletterTrait {
```

Comment:
Change this trait so that it has two methods: placeNewsletterSubscriptionBlock() and placeNewsletterUnsubscriptionBlock().
They both take an array of block settings as parameter. Inside the method, merge this array with a set of defaults.
Use shorter block id names, so it's less verbose.